src/warpedalloy/__init__.py

```diff
@@ -219,13 +218,13 @@ def sendOutFileDescriptor(self, fileDescriptor):
 
     def newSubProcess(self):
         """
-        
+
         """
         here, there = socketpair(AF_UNIX, SOCK_STREAM)
         owp = OneWorkerProtocol()
         serverTransport = UNIXServer(here, owp, None, None, 4321, self.reactor)
         owp.makeConnection(serverTransport)
-        argv = [sys.executable, __main__.__file__, b'w']
+        argv = [sys.executable, __file__, b'w']
```

Comment:
Why'd you change this from `__main__.__file__`, which re-runs the script that was run, to just `__file__`, which changes it to this file?